### ISSUE TITLE
[Feature Request] Provide support for selecting metric dimension

### ISSUE THREAD

**Issue by tomkerkhove** [microsoft/azure-load-testing#114]
**Is your feature request related to a problem? Please describe.**
Improve reported metrics in test details

**Describe the solution you'd like**
Provide support for selecting metric dimension so that the test metrics can be more scoped to the test target.

For example, when testing workloads on AKS the metrics are shared for all aspects of the cluster while the AKS metrics provide dimensions on which can be filtered (ie namespace)

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by Nagarjuna-Vipparthi** [comment]
@tomkerkhove Did you explore the 'Configure metrics' option on the test results page? This option gives you a way to customize the server-side metrics that are shown. 
https://learn.microsoft.com/en-us/azure/load-testing/how-to-monitor-server-side-metrics

Sharing some screenshots 
![image](https://user-images.githubusercontent.com/103505570/199890180-bf946c3d-a835-46d8-9f2b-f113fc056246.png)
![image](https://user-images.githubusercontent.com/103505570/199890203-df3fbc1e-ee32-4c5b-9d96-e8549e7bbfb1.png)


Please let us know if this helps

**Comment by tomkerkhove** [comment]
Yes I did, but the above is the metric namespace and not metric dimensions.

For example, App Service emits CPU % but you can filter it by instance. For Azure API Management and AKS this is important. Getting the used memory on a full AKS cluster vs just one namespace is a totally different view.

You can find all metrics and dimensions on https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported

**Comment by Nagarjuna-Vipparthi** [comment]
> Yes I did, but the above is the metric namespace and not metric dimensions.
> 
> For example, App Service emits CPU % but you can filter it by instance. For Azure API Management and AKS this is important. Getting the used memory on a full AKS cluster vs just one namespace is a totally different view.
> 
> You can find all metrics and dimensions on https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported

Got it. We will add this to our backlog and explore the best way to support this.

**Comment by tomkerkhove** [comment]
Any update on this?
